RWR and TieDIE Integration (#92)
Reed-CompBio/spras: docker-wrappers/TieDIE/README.md

# TieDIE Docker image

> A Docker image for [TieDIE](https://github.com/epaull/TieDIE) that is available on [DockerHub](https://hub.docker.com/r/erikliu24/tiedie).

Reviewer: ```suggestion
A Docker image for [TieDIE](https://github.com/epaull/TieDIE) that is available on [DockerHub](https://hub.docker.com/repository/docker/reedcompbio/tiedie).
```

To create the Docker image run:
```

> docker build -t erikliu24/tiedie -f Dockerfile .

Reviewer: ```suggestion
docker build -t reedcompbio/tiedie -f Dockerfile .
```

```
from this directory.

To inspect the installed Python packages:
```
winpty docker run erikliu24/tiedie pip list
```
The `winpty` prefix is only needed on Windows.

## Testing
Test code is located in `test/TieDIE`.
The `input` subdirectory contains test files `pathway.txt`, `target.txt` and `source.txt`.

> The Docker wrapper can be tested with `pytest` or a unit test with `pytest -k test_rwr.py`.

Reviewer: ```suggestion
The Docker wrapper can be tested with `pytest` or a unit test with `pytest -k test_tiedie.py`.
```

# TieDIE Docker image

A Docker image for [TieDIE](https://github.com/epaull/TieDIE) that is available on [DockerHub](https://hub.docker.com/r/erikliu24/tiedie).

To create the Docker image run:
```
docker build -t erikliu24/tiedie -f Dockerfile .
```
from this directory.

To inspect the installed Python packages:
```
winpty docker run erikliu24/tiedie pip list
```
The `winpty` prefix is only needed on Windows.

## Testing
Test code is located in `test/TieDIE`.
The `input` subdirectory contains test files `pathway.txt`, `target.txt` and `source.txt`.

> The Docker wrapper can be tested with `pytest`.

Reviewer 2: you could use pytest -k test_tiedie.py rather than just pytest (but I think either or can be written)
